loggroup-lambda-connector/src/loggroup-lambda-connector.js

```diff
@@ -5,13 +5,24 @@ async function sleep(waitTimeInMs) {
     return new Promise((resolve) => setTimeout(resolve, waitTimeInMs));
 }
 
-function subscribeToLambda(lambdaLogGroupName, lambdaArn, errorHandler) {
-    var params = {
-        destinationArn: lambdaArn,
-        filterName: 'SumoLGLBDFilter',
-        filterPattern: '',
-        logGroupName: lambdaLogGroupName
-    };
+function subscribeToLambda(lambdaLogGroupName, destinationArn, roleArn, errorHandler) {
```

Comment:
the function name is subscribeToLambda probably we can change it to subscribeLogGroupToService